styles/components/_menu.scss

```diff
@@ -1,60 +1,69 @@
 .page__container {
-  width: 100vw;
-  height: calc(100vh - 2rem);
-  background-color: white;
-  text-align: center;
+  z-index: -1;
+  height: 100vh;
+  background: url(/static/images/ui/background.png) no-repeat center;
+  background-size: cover;
```

Comment:
There is something wrong with positioning, I get a scroll on my screenand the START text is not centered

Reply:
Remove page-container class from menu-window

Reply:
I'll check positioning.